semcore/date-picker/src/components/InputTrigger.jsx

```diff
@@ -843,29 +656,160 @@ const MaskedInput = ({
     setAppliedWidth(width);
   }, [width, focused]);
 
+  const maskInstanceRef = React.useRef(null);
+  React.useEffect(() => {
+    if (!ref.current) return;
+    if (maskInstanceRef.current) return;
```

Comment:
`!maskInstanceRef.current` ?

Reply:
No, it's to prevent unexpected reinitialization